lib/driverProviders/sauce.ts

```diff
@@ -63,7 +63,8 @@ export class Sauce extends DriverProvider {
     this.config_.capabilities['username'] = this.config_.sauceUser;
     this.config_.capabilities['accessKey'] = this.config_.sauceKey;
     this.config_.capabilities['build'] = this.config_.sauceBuild;
-    let auth = 'https://' + this.config_.sauceUser + ':' + this.config_.sauceKey + '@';
+    let auth = (this.config_.sauceSeleniumAddressProtocol || 'https://') + this.config_.sauceUser +
```

Comment:
- adjusted auth part construction to use `sauceSeleniumUseHttp`
- decided not to switch to url/templates to keep changes to bare minimum